modules/storages/app/components/storages/project_storages/destroy_confirmation_dialog_component.html.erb

```diff
@@ -28,54 +28,68 @@ See COPYRIGHT and LICENSE files for more details.
 ++#%>
 
 <%=
-  primer_form_with(
-    model: @project_storage,
-    url: admin_settings_storage_project_storage_path(id: @project_storage),
-    data: {
-      controller: 'disable-when-checked',
-      'disable-when-checked-reversed-value': true
-    },
-    method: :delete
-  ) do |form|
-    render(Primer::Alpha::Dialog.new(
-      id: id,
-      title: nil,
-      test_selector: id,
-      size: :large
-    )) do |dialog|
-      dialog.with_body do
-        flex_layout do |layout|
-          layout.with_row(mb: 3) do
-            render(Primer::Beta::Heading.new(tag: :h1, text_align: :center)) do
-              render(Primer::Beta::Octicon.new(icon: :alert, color: :danger, size: :medium))
+  component_wrapper do
```

Comment:
This is just for semantics and convention, but since async dialogs replace stream content via JS, it's technically not necessary- which is why it worked before without it.